index.bs

```diff
@@ -505,9 +521,35 @@ The `:picture-in-picture` <a>pseudo-class</a> MUST match any <a>element</a>
 
 <em>This section is non-normative.</em>
 
-The API applies only to {{HTMLVideoElement}} in order to start on a minimal
-viable product that has limited security issues. Later versions of this
-specification may allow PIP-ing arbitrary HTML content.
+The {{HTMLVideoElement}} Picture-in-Picture API has limited security issues.
+
+## Restrictions on arbitrary content ## {#arbitrary-content-restrictions}
+
+The <a>Picture-in-Picture browsing context</a> MUST have top-level navigation
+blocked and the following <a>sandboxing flag set</a> applied:
+- <a>sandbox propagates to auxiliary browsing contexts flag</a>
+- <a>sandboxed auxiliary navigation browsing context flag</a>
+- <a>sandboxed document.domain browsing context flag</a>
```

Comment:
Link doesn't work.
It should be https://html.spec.whatwg.org/multipage/origin.html#sandboxed-document.domain-browsing-context-flag

Reply:
Done.